apps/benchmark/README.md

```diff
@@ -0,0 +1,17 @@
+# Performance benchmark
+
+This app runs a performance benchmark of different css styling implementations courtesy of [react-native-web](https://github.com/necolas/react-native-web/tree/master/packages/benchmarks).
+
+## Usage
+
+To start benchmarking run these commands from the root of the repo:
+
+```
+// Builds the app and watches for changes
+yarn nx run @griffel/benchmark:watch
+```
+
+```
+// Serves the benchmark app to use in the browser
+yarn nx run @griffel/benchmark:serve
+```
```

Comment:
````suggestion
```shell
# Builds the app and watches for changes
yarn nx run @griffel/benchmark:watch
```

```shell
# Serves the benchmark app to use in the browser
yarn nx run @griffel/benchmark:serve
```
````

Reply:
Done in [c3375d4](https://github.com/microsoft/griffel/pull/132/commits/c3375d4fc77d4ab50458d45b9de0eb35e1180609)